source/administration/config-settings.rst

```diff
@@ -309,6 +309,18 @@ Enable Team Directory
 | This feature's ``config.json`` setting is ``"EnableTeamListing": false`` with options ``true`` and ``false`` for above settings respectively.                        |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Default Channel Leave/Join Messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Allows for leave/join messages to be created in the default channel.
```

Comment:
Can we clarify a bit more about what this does? Are these team join leave messages in town square that are being enabled/disabled?